### ISSUE TITLE
Add the ability to generate javascript instead of coffeescript

### ISSUE THREAD

**Issue by JakeLaCombe** [kirtithorat/carrierwave-crop#4]
Hello!

Would it be possible to generate javascript instead of coffeescript code for the image uploader? I know of several developers that prefer javascript to coffeescript, and I feel being able to provide that option would help make the gem for flexible and easier to use.

**Comment by kirtithorat** [comment]
@JakeLaCombe Appreciate the suggestion, it would certainly be a bonus to provide a little flexibility.
The reason for preferring coffeescript over javascript for the gem was the adoption of coffeescript by Rails since version 3.1.

**Comment by DonGiulio** [comment]
I Agree with this request, I work on a rails 4 project that had support of coffeescript removed. so a javascript version of this would help me a lot too.
